pkgs/top-level/all-packages.nix

```diff
@@ -27629,6 +27629,7 @@ with pkgs;
         kdeIntegration = true;
         variant = "fresh";
       });
+    dolphinTemplates = true;
```

Comment:
I think this should now be passed to the wrapper?

Reply:
???? I dont understand what youre saying. Could you rephrase it?

Reply:
Oops, I mixed up things. The correct question is whether this should apply to both wrapped versions of LibreOffice

Reply:
Oh, yeah why not. My initial idea was that people who are using dolphin are most likely use choosing libreoffice-qt over libreoffice. But it won't hurt to have it enabled by default